### PR TITLE
Implement account backup screen.

### PR DESCRIPTION
**Pull request by abouolia**

### CODE REVIEW

File: packages/renderer/src/pages/Account/Backup/AccountBackup.tsx [hunk 1]
```diff
@@ -14,12 +14,20 @@
...
+import { Box, Stack } from "@mantine/core";
 import { AccountLayout } from "@templates/AccountLayout";
+import { AccountBackupDownload } from "./AccountBackupDownload";
+import { AccountRestoreDownload } from "./AccountBackupRestore";
 
 export function AccountBackup() {
   return (
     <AccountLayout>
-      <h1>Account Backup</h1>
+      <Box>
```
Comment (schowdhuri) [inline packages/renderer/src/pages/Account/Backup/AccountBackup.tsx:25]: Unnecessary `Box`

**Review by schowdhuri — CHANGES_REQUESTED** [review]
Mostly, looks good. Nice work with the language strings! Here are a couple of general code-style comments:

1. We must strike the right balance between simplicity and abstraction. `AccountBackupDownload` and `AccountRestoreDownload` are small and simple enough to be either expanded inline (preferred) or added to the same file. I see no good reason to create another layer of hierarchy here.
1. Create DOM elements only if necessary. I see a few redundant `Box` layers

**Comment by schowdhuri** [comment]
Closing because this is a work in progress

**Comment by abouolia** [comment]
> Edit

My reason about splitting every section into different files, maybe now has small code, but after backend integration will have another ~20/30 lines on the download or restore button handler espectialy if we did the download/restore in the renderer thread.

File: packages/renderer/src/pages/Account/Backup/AccountBackup.tsx (hunk 1 above)
Comment (abouolia) [reply packages/renderer/src/pages/Account/Backup/AccountBackup.tsx:25]: Removed

**Pull request closed without merging**